src/sql/workbench/contrib/notebook/browser/markdownToolbarActions.ts

```diff
@@ -26,19 +27,20 @@ export class TransformMarkdownAction extends Action {
 		tooltip: string,
 		private _cellModel: ICellModel,
 		private _type: MarkdownButtonType,
-		@INotebookService private _notebookService: INotebookService
+		@INotebookService private _notebookService: INotebookService,
+		@IInstantiationService private _instantiationService: IInstantiationService
 	) {
 		super(id, label, cssClass);
 		this._tooltip = tooltip;
 	}
 	public run(context: any): Promise<boolean> {
-		return new Promise<boolean>((resolve, reject) => {
+		return new Promise<boolean>(async (resolve, reject) => {
```

Comment:
Just get rid of the promise wrapping and make this an async function that just contains the code in the try

Reply:
Fixed